### ISSUE TITLE
Allow user model to be configurable

### ISSUE THREAD

**Issue by cjke** [Laravel-Backpack/Base#8]
At the moment, the user model is fixed at line 70 in the AuthController. Ideally it would be good to be able to define, say App\Models\Admin as the model.

**Comment by tabacitu** [comment]
Hi @cjke ,

First of all, sorry I haven't replied sooner - I guess some issues just slipped by me. Totally agree with this, I've included your suggestion in ```Backpack\Base v0.6.12``` (just pushed it). You'll notice this in your ```config/backpack/base.php``` file:

```php
    /*
    |--------------------------------------------------------------------------
    | User Model
    |--------------------------------------------------------------------------
    */

    // Fully qualified namespace of the User model
    'user_model_fqn' => '\App\User',
```

Cheers!

P.S. I'll go ahead and close this issue so we don't clog the section. Feel free to open it again if you encounter problems or comment if you have follow-ups.

**Comment by telmen** [comment]
it works only on registration but login. how would I do that to login?
